src/main/resources/templates/db_scripts/create_job_number_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS ${myuniversity}_${mymodule}.job_number (
+  last_number bigint NOT NULL DEFAULT 1,
+  type varchar(255) NOT NULL
+);
+INSERT INTO ${myuniversity}_${mymodule}.job_number (last_number, type)
+SELECT 1, 'Logs'
+WHERE NOT EXISTS (SELECT * FROM ${myuniversity}_${mymodule}.job_number);
```

Comment:
Adding WHERE can help with migration in case when we have multiple types
SELECT 1, type 
WHERE NOT EXISTS (SELECT * FROM ${myuniversity}_${mymodule}.job_number WHERE type = 'FUND_BULK_UPDATES_LOGS');

Reply:
updated SQL, also new script checked it will not affect to another type